### ISSUE TITLE
Exporting marblenet to onnx doesn't produce the same outputs as torch model

### ISSUE THREAD

**Issue by naymaraq** [NVIDIA/NeMo#3974]
Hi NeMo team,

I am trying to convert marblnet to onnx format. Compared to torch outputs, onnx outputs are slightly different. 
Did you check that onnx and torch outputs are exactly the same?

**Comment by naymaraq** [comment]
I manage to find the problem with cause different outputs. I have two python environments. One of them is created with reinstall.sh and one of them I created by myself with exact versions of torch, torchaudio, numpy and librosa. But the output of torchaudio.transforms.MFCC is different (which then leads to different logits).

So, any clues about why MFCC is working differently in different enviroments with exact same versions?

**Comment by EmreOzkose** [comment]
I have same issue with titanet-l model. Tİtanet-l requires to calculate Mels, but since Mels is not exportable object in Nemo, onnx model didn't contain this step and I had to calculate before. I calculated with this:

```
from nemo.collections.asr.modules.audio_preprocessing import (
        AudioToMelSpectrogramPreprocessor,
    )
    preprocessor = AudioToMelSpectrogramPreprocessor(features=80,
                                                     n_fft=512,
                                                     window_size=0.025,
                                                     window_stride=0.01,
                                                     window="hann",
                                                     dither=1e-5,)
    processed_signal, processed_signal_len = preprocessor(
            input_signal=signal, length=length,
        )
```
where I find parameter values by debugging actual inference part. There is also `nemo.collections.asr.modules.AudioToMFCCPreprocessor`. I think that it is also important how to export onnx model, there are 2 way: model.export("onnx_model.onnx") or [export.py](https://github.com/NVIDIA/NeMo/blob/main/scripts/export.py).

**Comment by EmreOzkose** [comment]
I realize that calculated mels are different. If someone can share full params to calculate Mels and MFCCs for diarization and VAD model, it will be very helpfull.

**Comment by titu1994** [comment]
Export.py uses the model.export() underneath the hood, there should be no major difference. We don't support exporting of the Mel spectrogram preprocessor, we may consider it in the long term future.
